package.json

```diff
@@ -1,24 +1,41 @@
 {
-  "name": "tigris-nextjs13-starter-kit",
+  "name": "tigris-vercel-starter",
```

Comment:
There is already a package with this name. Let's call this one `tigris-nextjs13-starter-kit`